.editorconfig

```diff
@@ -0,0 +1,13 @@
+[*]
+indent_style = space
+indent_size = 4
```

Comment:
Should we vote on the indentation? I could live with both 4 and 2, with a preference for 4 (eyes are getting worse when ageing :)

Reply:
I think 4 makes most sense because it is in-line with what is use in Perl code as well and probably also what we already use in most shell files in most places.

Reply:
having `set shiftwidth=4 tabstop=4 softtabstop=4 shiftround expandtab` in my vimrc I strongly prefer 4 😄

Reply:
Also note: we use 2 spaces in yaml files (so that should be probably also included in this configuration somehow).